### ISSUE TITLE
How to migrate to Ember Data 1.13 new serializer api

### ISSUE THREAD

**Issue by Gaurav0** [embersherpa/ama#22]
Please discuss the differences between the old serializer api and the new serializer api and discuss how to migrate a model from the old api to the new api.

**Comment by taras** [comment]
@Gaurav0 I didn't see any change in the model, but the serializer is quiet different. Are you using a RESTSerializer?

**Comment by Gaurav0** [comment]
Sorry, should have said "migrate a serializer".

I'm going to move Ember Twiddle first as I will be able to do my learning (and ask for help) in public. :) Ember Twiddle is using a Custom Serializer that extends from JSONSerializer. I have applications that use RESTSerializer though.

The real issue is that the long planned 1.13 transition guide for Ember Data serializers was never published (even though 2.0 is now out).

**Comment by taras** [comment]
> I'm going to move Ember Twiddle first as I will be able to do my learning (and ask for help) in public. :) 

Great idea :)

> The real issue is that the long planned 1.13 transition guide for Ember Data serializers was never published (even though 2.0 is now out).

I was using this article http://emberjs.com/blog/2015/06/18/ember-data-1-13-released.html when doing an upgrade. The hardest part was knowing what JSONAPI format was expected because JSONAPI example were very limited.

**Comment by frank06** [comment]
Far from being a complete guide, but I wrote [this article](http://emberigniter.com/fit-any-backend-into-ember-custom-adapters-serializers/) as some sort of extension of the aforementioned "Ember Data 1.13 released". I found myself going back to that Ember Data blog post a million times!   I think better docs regarding JSON API are coming soon :)

**Comment by Gaurav0** [comment]
@frank06 Yes I saw that two weeks ago. It truly helped and finally enabled me to get Ember Twiddle into the new serializer api. However, for the most part I was still guessing and testing if random stuff worked rather than gaining any sort of understanding.

**Comment by frank06** [comment]
@Gaurav0 Awesome, I didn't know you were the great force behind Ember Twiddle. I love it.
